src/main/groovy/com/entagen/jenkins/JenkinsJobManager.groovy

```diff
@@ -68,6 +69,9 @@ class JenkinsJobManager {
             if (startOnCreate && missingJob.jobName.contains("Rev.com-featurebranch-build-feature")) {
                 jenkinsApi.startJob(missingJob)
             }
+            if (isUnitTestOnly && missingJob.jobName.contains("Rev.com-featurebranch-deploy-feature")) {
```

Comment:
The templates start off disabled. I think it would better to not enable them in the first place. I would make the enableJob on line 68 conditional on 

```
if (!isUnitTestOnly || !missingJob.jobName.contains("Rev.com-featurebranch-deploy-feature")) {
    jenkinsApi.enableJob(missingJob.jobName)
}
```